python/servo/build_commands.py

```diff
@@ -320,6 +320,44 @@ def build(self, target=None, release=False, dev=False, jobs=None,
                 for ssl_lib in ["libcryptoMD.dll", "libsslMD.dll"]:
                     shutil.copy(path.join(env['OPENSSL_LIB_DIR'], "../bin" + msvc_x64, ssl_lib),
                                 servo_exe_dir)
+                vs_dll_dir = None
+                vs_platform = os.environ.get("PLATFORM", "").lower()
+                vc_dir = os.environ.get("VCINSTALLDIR", "")
+                vs_version = os.environ.get("VisualStudioVersion", "")
+                msvc_deps = [
+                    "api-ms-win-crt-runtime-l1-1-0.dll",
+                    "msvcp140.dll",
+                    "vcruntime140.dll",
+                ]
+                # Check if it's Visual C++ Build Tools or Visual Studio 2015
+                vs14_vcvars = path.join(vc_dir, "vcvarsall.bat")
+                is_vs14 = True if os.path.isfile(vs14_vcvars) or vs_version == "14.0" else False
+                if is_vs14:
+                    vs_dll_dir = path.join(vc_dir, "redist", vs_platform, "Microsoft.VC140.CRT")
+                elif vs_version == "15.0":
+                    redist_dir = os.environ["VCToolsRedistDir"]
+                    # there are two possible paths `x64\Microsoft.VC150.CRT` or `onecore\x64\Microsoft.VC150.CRT`
+                    redist1 = path.join(redist_dir, vs_platform, "Microsoft.VC150.CRT")
+                    redist2 = path.join(redist_dir, "onecore", vs_platform, "Microsoft.VC150.CRT")
+                    if os.path.isdir(redist1):
+                        vs_dll_dir = redist1
+                    elif os.path.isdir(redist2):
+                        vs_dll_dir = redist2
+                if vs_version == "15.0" or is_vs14:
```

Comment:
I'm not sure, but I think `if vs_dll_dir:` may have better semantics here.
Also, if these deps are required, it would be nice to have some error handling in case we can't find the `vs_dll_dir`.